pkgs/development/libraries/libpng/default.nix

```diff
@@ -25,14 +25,14 @@ in stdenv.mkDerivation rec {
 
   propagatedBuildInputs = [ zlib ];
 
-  doCheck = stdenv.hostPlatform == stdenv.buildPlatform;
+  doCheck = true;
```

Comment:
This will be still skipped for cross-builds, right?

Reply:
Yes, that's my understanding! :)